docs/misc/faq.md

```diff
@@ -135,8 +135,7 @@ thousands or even millions of containers running in parallel.
 
 ### How do I connect Docker containers?
 
-Currently the recommended way to link containers is via the link primitive. You
-can see details of how to [work with links here](../userguide/dockerlinks.md).
+Currently the recommended way to connect containers is via the Docker network. You can see details of how to [work with links here](https://docs.docker.com/networking).
```

Comment:
Link caption still refers to "work with links", should this be "Docker network", or something?

Are we referring to this feature as "the Docker network"? It almost sounds like a commercial product / hosted solution. (could be just me)